octoprint_mrbeam/static/js/helpers/working_area_helper.js

```diff
@@ -1,4 +1,48 @@
 class WorkingAreaHelper {
+    static versionCompare(v1, v2, options) {
+        var lexicographical = options && options.lexicographical,
+            zeroExtend = options && options.zeroExtend,
```

Comment:
same here... We could rename this to isStrictComparison. on the documentation we could then explain what extrict means. which is 2.1 != 2.1.0 and not strict will do 2.1 == 2.1.0.

Reply:
we actually changed the whole handling. Will send you a new review request once done.